polls/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from .models import Annotation
+
+
+class AnnotationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Annotation
+        fields = "__all__"
```

Comment:
It's okay to use all available fields in a serializer configuration, but listing them out is handier in terms of clarity and in case there's a change or restriction to be done, not really a big deal but it's good practice.